web/Views/Admin/AllProperties.cshtml

```diff
@@ -0,0 +1,102 @@
+@model PropertiesDashboardViewModel
+
+@{
+    ViewData["Title"] = "All Properties";
+}
+
+<h2>All properties</h2>
+
+<div>
+    <div>
+        <div asp-validation-summary="All" class="text-danger"></div>
+            @{
+                var totalRow = 0;
+                var propPerTab = 10;
+            }
+            @for (int tab = 0; tab <= Model.Properties.Count / propPerTab; tab++)
```

Comment:
This is paginating it client side, which is to say all the properties are sent to the browser, and then we're using javascript to only show some of them at once.

Server side pagination would be to only send the current page to the browser, and then make another request when we want a different page.

Let me know if that doesn't make sense ^

Client side pagination works well for up to a few hundred records since switching tabs is instant, but past that we should use server side. If we're really going to have thousands of properties like ChangeAhead say, then we'll want this to be paginated server side at some point. You can do that now if you'd like, but otherwise lets merge this and raise another ticket to come back to it.